kuma/users/jinja2/users/ban_user_and_cleanup.html

```diff
@@ -0,0 +1,61 @@
+{% extends "base.html" %}
+
+{% block body_attributes %}{% endblock %}
+
+{% block bodyclass %}user ban-user{% endblock %}
+
+{% block title %}{{ _('Ban') }} {{ detail_user }}{% endblock %}
+{% set styles = ('dashboards', 'jquery-ui') %}
+{% set classes = 'compare' %}
+
+{% block site_css %}
+  {{ super() }}
+  {% stylesheet 'users' %}
+  {% stylesheet 'user-banned' %}
+{% endblock %}
+
+{% block content %}
+<section id="content">
+{# TODO: Add an if block if the user has already been identified as a spammer #}
+
+{% if not user_banned %}
+  <h1>{{ _('Identify') }} <a href="{{ detail_user.get_absolute_url() }}">{{ detail_user }}</a> {{ _('as a spammer?') }}</h1>
+{% else %}
+  <h1><a href="{{ detail_user.get_absolute_url() }}">{{ detail_user }}</a> has already been banned. You may review their revisions below.</h1>
+{% endif %}
+
```

Comment:
It seems this ``TODO`` is already done - there's an ``if``/``else`` block with a different message if the user is already banned.  Should the comment be removed, or is there another missing block?